File: 1903. Largest Odd Number in String/solution.java
class Solution {
  public String largestOddNumber(String num) {
    for (int index = num.length() - 1; index > -1; index--){
      int value = num.charAt(index) - '0';
      if (value % 2 == 1){
        return num.substring(0, index + 1);
      }
    }

    return "";
  }
}

File: 1903. Largest Odd Number in String/solution.js
/**
 * @param {string} num
 * @return {string}
 */
 var largestOddNumber = function(num) {
  for (let index = num.length - 1; index > -1; index--){
    let value = parseInt(num[index]);
    if (value % 2 == 1){
      return num.slice(0, index + 1);
    }
  }

  return "";
};